pkg/controllers/cloud_config_sync_controller.go

```diff
@@ -89,6 +99,18 @@ func (r *CloudConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	return ctrl.Result{}, nil
 }
 
+func (r *CloudConfigReconciler) isCloudConfigSyncNeeded(platformStatus *configv1.PlatformStatus) (bool, error) {
+	if platformStatus == nil {
+		return false, fmt.Errorf("platformStatus is required")
+	}
+	switch platformStatus.Type {
+	case configv1.AWSPlatformType: // aws ccm does not use cloud-config at the moment
+		return false, nil
+	default:
+		return true, nil
+	}
```

Comment:
Are we sure we want to make this an exclusive list rather than an inclusive list? How many platforms do vs don't use the cloud config?

Reply:
Only AWS at the moment